docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Reactive_Messsaging_SmallRye.adoc

```diff
@@ -289,6 +291,72 @@ While we do expose the Kafka Clients jar in our BOMs, its usage is limited to
 ** `org.apache.kafka.common.serialization.Serializer`
 ** Implementatations of `org.apache.kafka.common.serialization.Deserializer` and `org.apache.kafka.common.serialization.Serializer` in the `org.apache.kafka.common.serialization` package
 
+// New
+==== AMQP Connector
+
+An example of a minimal `microprofile-config.properties` file for AMQP for the example application shown previously:
+
+```
+amqp-host=localhost
+amqp-port=5672
+amqp-username=artemis
+amqp-password=artemis
+
+mp.messaging.outgoing.to.connector=smallrye-amqp
+mp.messaging.outgoing.to.address=my-topic
+
+mp.messaging.incoming.from.connector=smallrye-amqp
+mp.messaging.incoming.from.address=my-topic
+```
+
+Next we will briefly discuss each of these entries. Remember the `to` channel is on the `send()` method, and the `from` channel is on the `receive()` method.
+
+
+'amqp-host=localhost' '`amqp-port=5672` points the connector to an AMQP broker running on `localhost:5672`. As before we could also have done these for an individual channel by for example specifying  `mp.messaging.outgoing.to.host=localhost` instead. If the host is not specified, it defaults to `localhost.
```

Comment:
@kabir This can be done in a follow up fix, but whatever turns on italics here needs to be fixed.

Note -- please don't make any docs fixes I mention here on this PR unless you have to change something that justifies new CI. They can come later.

Reply:
@bstansberry I've never seen the source be italic before, but it turns out it was due to some unclosed quotes

Reply:
Done in https://github.com/wildfly/wildfly/pull/17491